### DISCUSSION TITLE
Support for tagged template literal (like emotion.css)

### DISCUSSION THREAD

**Discussion by younes0** [chakra-ui/panda#880] [General]
Quick question: the CHANGELOG mentions  "Add support for tagged template literal version.  This features is pure css approach to writing styles, and can be a great way to migrate from styled-components and emotion."

Does it means we will be able to use emotion.css literal / css prop ?
```tsx
 <div css={style} />
 
  const style = css`
    padding: 32px;
    &:hover {
      color: ${color};
    }
  `;
  ```

Currently, emotion.js is the only library that prevents us to migrate to App Router: https://github.com/emotion-js/emotion/issues/2978

**Comment by anubra266** [comment]
@younes0 yes that's what it means
